### PR TITLE
Add Panel and Pill components

### PR DESCRIPTION
**Pull request by rppld**
http://localhost:9001/?path=/story/panel--single
http://localhost:9001/?path=/story/panel--group
http://localhost:9001/?path=/story/pill--primary

### CODE REVIEW

File: src/components/Panel/index.stories.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+export const Group = () => (
+  <Panel>
+    <PanelContent>
+      <PanelBody>
+        <H4>Title</H4>
+        <PanelText>Lorem ipsum dolor sit amet</PanelText>
+        <BaseButton>Edit</BaseButton>
+      </PanelBody>
+      <PanelFooter>
```
Comment (glenngijsberts) [inline src/components/Panel/index.stories.js:34]: Footer? 😛

**Comment by glenngijsberts** [comment]
Don't we need to export this component in `/src/index.js`?

**Review by glenngijsberts — APPROVED** [review]
Looks 🔥

File: src/components/Panel/index.stories.js (hunk 1 above)
Comment (rppld) [reply src/components/Panel/index.stories.js:34]: Haha yeahhh, I know. Been thinking about that one for quite a while. Had `PanelAdornment` at first, but it’s not really an "adornment". I think it makes more sense if you look at it this way:

Screenshot 2019-09-26 at 20 44 53

File: src/components/Pill/index.stories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export default {
+  title: 'Pill',
+}
+
+export const Primary = () => <Pill>Primary</Pill>
+export const Secondary = () => <Pill variant="secondary">Secondary</Pill>
+export const Success = () => <Pill variant="success">Success</Pill>
+export const Danger = () => <Pill variant="danger">Danger</Pill>
+export const Warning = () => <Pill variant="warning">Warning</Pill>
```
Comment (rppld) [inline src/components/Pill/index.stories.js:13]: Add an example with icon and double-check alignment.

**Pull request merged**